src/Pages/home/Components/Title.tsx

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import Box from "@material-ui/core/Box";
+import Button from "@material-ui/core/Button";
+import Typography from "@material-ui/core/Typography";
+import { createStyles, makeStyles } from "@material-ui/core/styles";
+import brainImg from "../../../assets/brain_welcome.png";
+import AddIcon from "../../../assets/add.png";
+
+//todo Style
+const useStyle = makeStyles(() =>
+  createStyles({
+    root: {
+      flexGrow: 1,
+    },
+
+    //! Box
+    boxWelcome: {
+      width: "616px", //decrease 15%
+      height: "147px",
+      marginTop: "48px",
+      marginLeft: "32px",
+      borderRadius: "20px",
+      background:
+        "linear-gradient(94.59deg, #5028C6 9.91%, #FD85FF 81.34%, #F40000 111.1%)",
+      boxShadow: "0px 2px 4px rgba(0, 0, 0, 0.1)",
+    },
+    boxTotalPatients: {
+      width: "185px",
+      height: "147px",
+      marginTop: "48px",
+      marginLeft: "24px",
+      background: "#FFFFFF",
+      boxShadow: "0px 2px 4px rgba(0, 0, 0, 0.1)",
+      borderRadius: " 20px",
+    },
+    //! Text
+    textWelcome: {
+      marginLeft: "23px",
```

Comment:
All of text or whatever are child components (Ex. text in button, text in box, box in box and so on).  I suggest using padding rules on parent components more than margin rules on all child components.

Reply:
include this line, line 42,43